cms/djangoapps/contentstore/views/library.py

```diff
@@ -60,6 +62,9 @@ def library_handler(request, library_key_string=None):
 
     # request method is get, since only GET and POST are allowed by @require_http_methods(('GET', 'POST'))
     if library_key_string:
+        # Revoke sudo privileges from a request explicitly
```

Comment:
@dan-f Good catch, I did it at wrong place we need to revoke it in ```_display_library``` instead here because in ```_display_library```  we are displaying xblocks.